docs/current/guides/snippets/ci/go/circle.yml

```diff
@@ -8,8 +8,11 @@ jobs:
       - setup_remote_docker:
           docker_layer_caching: true
       - run:
-          name: Dagger Pipeline
-          command: go run main.go
+          name: Install Dagger CLI
+          command: cd /usr/local && { curl -L https://dl.dagger.io/dagger/install.sh | sh; cd -; }
+      - run:
+          name: Run Dagger pipeline
+          command: dagger run go run main.go
```

Comment:
```suggestion
          command: dagger run --progress plain go run main.go
```

Reply:
@kpenfound do we need this? My understanding is that the CLI automatically switches to `plain` progress when running on a non-terminal UI. This was also my experience with GitHub/GitLab/etc. @vito to confirm.

Reply:
Normally that works, yeah, but for some reason it wasn't working in CircleCi and I had to add `--progress plain`. Are you able to see this run? https://app.circleci.com/pipelines/github/kpenfound/greetings-api/87/workflows/07338dc8-97b7-4bd4-8019-b885cb0dd768/jobs/98

Reply:
Yes, I see it. I wonder if @vito has any idea why the behaviour is different on CircleCI vs. GitHub/GitLab?

Reply:
My guess is CircleCI actually runs the steps with a TTY attached, whereas the others direct stdio to a regular file or pipe. When a TTY is attached the CLI will default to the fancy TUI, since that's supposed to mean it's rendering in a terminal. I wonder if CircleCI gives some other indication that the CLI should use plain format. Could be worth dumping the env vars, maybe we can detect based on `$TERM` or `$CI` or something.

@kpenfound It looks like it was actually able to render the TUI, but failed to set up the engine. Are the two related? i.e. setting the progress format to `plain` fixes the engine issue?

Reply:
Yeah in that run it failed the engine due to the environment. Separate issue. Basically what you see while it's running is just the last line of what the TUI would be showing, it's weird.